docs/install_guide.adoc

```diff
@@ -0,0 +1,206 @@
+operatorgroup.operators.coreos.com/openstack unchanged
+subscription.operators.coreos.com/watcher-operator created
+----
++
+. Check that the operator is installed:
++
+[,console]
+----
+$ oc get subscription.operators.coreos.com/watcher-operator -n openstack-operators
+NAME               PACKAGE            SOURCE                   CHANNEL
+watcher-operator   watcher-operator   watcher-operator-index   alpha
+
+$ oc get pod -l openstack.org/operator-name=watcher -n openstack-operators
+NAME                                                  READY   STATUS    RESTARTS   AGE
+watcher-operator-controller-manager-dd95db756-kslw9   2/2     Running   0          44s
+----
+
+== Deploying the Watcher Service
+
+Now, you will need to create a Watcher Custom Resource based on the `Watcher CRD`.
+
+.Procedure
+
+. Use the following commands to _view_ the `Watcher CRD` definition and specification schema:
++
+[,console]
+----
+$ oc describe crd watcher
+
+$ oc explain watcher.spec
+----
++
+. Add a `WatcherPassword` field to the `Secret` created as part of the ControlPlane deployment as explained in https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-RHOCP-for-RHOSO#proc_providing-secure-access-to-the-RHOSO-services_preparing[Providing secure access to the Red Hat OpenStack Services on OpenShift services]. Then `oc apply` the updated file and check the `WatcherPassword` is included.
++
+[,console]
+----
+$ oc apply -f <secret file> -n openstack
+
+$ oc describe secret osp-secret -n openstack | grep Watcher
+WatcherPassword:                  9 bytes
+----
++
+. Create a file on your workstation named `watcher.yaml` to define the Watcher CR. Your file should contain parameters similar to the example below:
++
+[source,yaml]
+----
+apiVersion: watcher.openstack.org/v1beta1
+kind: Watcher
+metadata:
+  name: watcher
+spec:
+  databaseInstance: "openstack"
+  secret: <name of the secret with the credentials of the ControlPlane deploy>
+  tls:
+    caBundleSecretName: "combined-ca-bundle"
+----
++
+For futher context,
+https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/deploying_red_hat_openstack_services_on_openshift/assembly_creating-the-control-plane#proc_creating-the-control-plane_controlplane[Creating the control plane] references similar commands and procdures to explain how to define an OpenStackControlPlane custom resource (CR).
++
+. `oc apply` to configure Watcher
++
+[,console]
+----
+$ oc apply -f watcher.yaml -n openstack
+watcher.watcher.openstack.org/watcher configured
+----
++
+. To check if the service status, run:
++
+[,console]
+----
+$ oc wait -n openstack --for condition=Ready --timeout=300s Watcher watcher
+watcher.watcher.openstack.org/watcher condition met
+----
++
+where `Watcher` refers to the _kind_ and `watcher` refers to the name of the CR.
```

Comment:
Wow. `Watcher` and `watcher` is very similar. Would it make sense to change the name of the CR to something else? I assume that the kind is fixed and must be `Watcher`?

Reply:
This pattern is common with the rest of operators, the default `Nova` CR name is `nova`, `Cinder` is `cinder` and so on. IMO this makes it easier and more predictable. CR names are case insensitive in most CLI usages although the Kind in yaml needs to be Watcher.